ts-web-component/src/types.ts

```diff
@@ -1,4 +0,0 @@
-export type TodoItemElement = HTMLInputElement & {
```

Comment:
It was not an input element

but keep the types together with their implementations